packages/openapi-v3-types/README.md

```diff
@@ -0,0 +1,52 @@
+# @loopback/openapi-spec
+
+TypeScript type definitions for OpenAPI Spec/Swagger documents.
+
+## Overview
+
+TypeScript definitions describing the schema of OpenAPI/Swagger documents,
+including LoopBack-specific extensions.
+
+_@jannyHou: will add more doc here to specify the version of types after we decide how to support new versions/keep old versions_
```

Comment:
I think this comment is no longer relevant?

Reply:
@strongloop/loopback-next discussed with @bajtos , we think it's good to keep swagger 2.0 specification types and OpenAPI 3.0.0 specification types in two package, and create new package for any major version upgrade from OpenAPI, any opinions/objections?

I will remove the comment from doc if we agree to go with multiple packages for those spec types.

Reply:
We should split those up into separate modules, yes, since they're separate specifications. :)